Pensel/firmware/applications/pensel_v2.c

```diff
@@ -77,6 +94,33 @@ void clear_critical_errors(void) {
     #endif
 }
 
+extern __IO uint8_t Receive_Buffer[64];
+extern __IO uint32_t Receive_length;
```

Comment:
There seems to be a lot of inconsistencies with variable naming standards, such as `Aaaa_Aaaa`, `Aaaa_aaaa`, `aaaa_aaaa`. Worth looking through the code and trying to standardize to make everything more cohesive

Reply:
Unfortunately due to my standard clashing with STM’S HAL naming convention. Definitely will clean up the main code areas